OracleGoldenGate/README.md

```diff
@@ -0,0 +1,98 @@
+
+To build the images, use the [buildDockerImage.sh](buildDockerImage.sh) script.
+
+### Building Oracle GoldenGate Docker Images
+**IMPORTANT:** You must download the installation binaries of Oracle GoldenGate. You only need to provide the binaries for the version you plan to install. The binaries can be downloaded from the [Oracle Technology Network](http://www.oracle.com/technetwork/middleware/goldengate/downloads/index.html). You also must have Internet connectivity when building the Docker image for the package manager. Note that you must not uncompress the Oracle GoldenGate binaries. The script will handle that for you and fail if you uncompress them manually!
+
+Once you have downloaded the Oracle GoldenGate software, run the **`buildDockerImage.sh`** script:
+
+    [oracle@localhost dockerfiles]$ ./buildDockerImage.sh -h
+    Oracle GoldenGate distribution ZIP file not specified.
+
+    Usage: ./buildDockerImage.sh [-h | <ogg-zip-file-name>] [<docker-build-options> ...]
+    Where:
+      ogg-zip-file-name       Name of OGG ZIP file
+      docker-build-options    Command line options for Docker build
+
+    Example:
+      buildDockerImage.sh ~/Downloads/fbo_ggs_Linux_x64_shiphome.zip --no-cache
+
+**IMPORTANT:** The result will be a Docker image with the Oracle GoldenGate binaries installed. Created images will follow the naming convention of **oracle/goldengate-&lt;edition&gt;:&lt;version&gt;**, for example:
+
+- `oracle/goldengate-standard:12.2.0.1.1`
+- `oracle/goldengate-standard:12.3.0.1.0`
+- `oracle/goldengate-microservices:12.3.0.1.0`
+
+#### Changing the Base Image
+By default, the base image used when building Oracle GoldenGate Docker images is `container-registry.oracle.com/database/instantclient:12.2.0.1`.  This value can be changed by setting the environment variable `BASE_IMAGE` when executing the `buildDockerImage.sh` script. For example:
+
+    BASE_IMAGE="container-registry.oracle.com/database/enterprise" ./buildDockerImage.sh ~/Downloads/fbo_ggs_Linux_x64_shiphome.zip
+
+### Running Oracle GoldenGate in a Docker container
+
+To run your Oracle GoldenGate Docker image use the **docker run** command as follows:
+
+    docker run --name <container name> \
+        -e OGG_SCHEMA=<schema for OGG> \
+        -e OGG_ADMIN=<admin user name> \
+        -e OGG_ADMIN_PWD=<admin password> \
+        -e OGG_DEPLOYMENT=<deployment name for Microservices Architecture> \
+        -v [<host mount point>:]/u02/ogg \
+        -v [<host mount point>:]/u02/ogg/var/data \
+        oracle/goldengate-microservices:12.3.0.1.0
+
+    Parameters:
+       --name:        The name of the container (default: auto generated)
+       -e OGG_SCHEMA: The GGSCHEMA to use for OGG (default: `oggadmin`)
+       -e OGG_ADMIN:  The name of the administrative account to create for Microservices Architecture (default: `oggadmin`)
+       -e OGG_ADMIN_PWD:
+                      The password for the administrative account (default: value of `OGG_ADMIN`)
```

Comment:
The Standard Edition does not use passwords.  The only way to use GoldenGate Standard Edition is by running the `ggsci` utility from within a running container.